sdk/python/kfp_tekton/compiler/compiler.py

```diff
@@ -195,8 +194,9 @@ def _create_pipeline_workflow(self, args, pipeline, op_transformers=None, pipeli
 
       workflow = workflow + [pipelinerun]
 
-    # Use regex to replace all the Argo variables to Tekton variables.
-    # For variables that are unique to Argo, we prompt a warning to warn users on the non-supported variables.
+    # Use regex to replace all the Argo variables to Tekton variables. For variables that are unique to Argo,
+    # we prompt a warning to warn users on the non-supported variables. Here is the list of Argo variables.
```

Comment:
"`# we raise an Error to alert users about the unsupported`" ... :-)

Reply:
thanks christian.